plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/BaseDeltaLakeMinioConnectorTest.java

```diff
@@ -1063,16 +1063,39 @@ public void testAlterTableWithUnsupportedProperties()
 
         assertUpdate("CREATE TABLE " + tableName + " (a_number INT)");
 
-        assertQueryFails("ALTER TABLE " + tableName + " SET PROPERTIES change_data_feed_enabled = true",
-                "The following properties cannot be updated: change_data_feed_enabled");
-        assertQueryFails("ALTER TABLE " + tableName + " SET PROPERTIES change_data_feed_enabled = true, checkpoint_interval = 10",
-                "The following properties cannot be updated: change_data_feed_enabled, checkpoint_interval");
+        assertQueryFails("ALTER TABLE " + tableName + " SET PROPERTIES checkpoint_interval = 10",
+                "The following properties cannot be updated: checkpoint_interval");
         assertQueryFails("ALTER TABLE " + tableName + " SET PROPERTIES writer_version = 4, partitioned_by = ARRAY['a']",
                 "The following properties cannot be updated: partitioned_by");
 
         assertUpdate("DROP TABLE " + tableName);
     }
 
+    @Test
+    public void testSettingChangeDataGeedEnabledProperty()
+    {
+        String tableName = "test_enable_and_disable_cdf_" + randomNameSuffix();
+        assertUpdate("CREATE TABLE " + tableName + " (page_url VARCHAR, domain VARCHAR, views INTEGER)");
+        assertThat((String) computeScalar("SHOW CREATE TABLE " + tableName)).contains("writer_version = 2");
+
+        assertQueryFails("ALTER TABLE " + tableName + " SET PROPERTIES change_data_feed_enabled = true",
+                "writer_version is set to less than 4 which is a minimal writer_version that supports change_data_feed_enabled");
+
+        assertQueryFails("ALTER TABLE " + tableName + " SET PROPERTIES change_data_feed_enabled = false",
+                "writer_version is set to less than 4 which is a minimal writer_version that supports change_data_feed_enabled");
+
+        assertUpdate("ALTER TABLE " + tableName + " SET PROPERTIES writer_version = 4");
+
+        assertUpdate("ALTER TABLE " + tableName + " SET PROPERTIES change_data_feed_enabled = true");
+        assertThat((String) computeScalar("SHOW CREATE TABLE " + tableName)).contains("change_data_feed_enabled = true");
+
+        assertUpdate("ALTER TABLE " + tableName + " SET PROPERTIES change_data_feed_enabled = false");
+        assertThat((String) computeScalar("SHOW CREATE TABLE " + tableName)).contains("change_data_feed_enabled = false");
+
+        assertUpdate("ALTER TABLE " + tableName + " SET PROPERTIES change_data_feed_enabled = true");
+        assertThat((String) computeScalar("SHOW CREATE TABLE " + tableName)).contains("change_data_feed_enabled = true");
+    }
```

Comment:
can we verify `writer_version` as well here along with `change_data_feed_enabled`?